Sources/Plasma/Apps/plClient/plClient.cpp

```diff
@@ -373,6 +377,58 @@ bool plClient::Shutdown()
     return false;
 }
 
+void plClient::InitDLLs() {
+    hsStatusMessage("Init dlls client\n");
+
+    std::vector<plFileName> dlls = plFileSystem::ListDir("ModDLL",
+#if defined(HS_BUILD_FOR_WIN32)
+        "*.dll"
+#elif defined(HS_BUILD_FOR_APPLE)
+        "*.dylib"
+#else
+        "*.so"
+#endif
+    );
+
+    for (auto iter = dlls.begin(); iter != dlls.end(); ++iter)
+    {
+#ifdef HS_BUILD_FOR_WIN32
+        hsLibraryHndl mod = LoadLibraryW(iter->WideString().data());
+#else
+        hsLibraryHndl mod = dlopen(iter->AsString().c_str(), RTLD_LAZY | RTLD_LOCAL);
+#endif
+
+        if (mod)
+        {
+#ifdef HS_BUILD_FOR_WIN32
+            pInitGlobalsFunc initGlobals = (pInitGlobalsFunc)GetProcAddress(mod, "InitGlobals");
+#else
+            pInitGlobalsFunc initGlobals = (pInitGlobalsFunc)dlsym(mod, "InitGlobals");
```

Comment:
Can we use explicit `static_cast` or `reinterpret_cast` here? (I haven't checked which one is correct here - it might need to be `reinterpret_cast` for the conversion from `void*` to a function pointer...)